app/controllers/form_controller.rb

```diff
@@ -109,11 +106,19 @@ def responses_for_page(page)
   end
 
   def find_resource
-    @lettings_log = current_user.lettings_logs.find_by(id: params[:id])
+    @log = if params.key?("sales_log")
+             current_user.sales_logs.find_by(id: params[:id])
+           else
+             current_user.lettings_logs.find_by(id: params[:id])
+           end
   end
 
   def find_resource_by_named_id
-    @lettings_log = current_user.lettings_logs.find_by(id: params[:lettings_log_id])
+    @log = if params["sales_log_id"].present?
```

Comment:
```suggestion
    @log = if params[:sales_log_id].present?
```

Does this need to be a string?